src/components/molecules/MenuPopoverContent/MenuPopoverContent.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import {
+  HOMEPAGE_URL,
+  PRIVACY_POLICY,
+  TERMS_AND_CONDITIONS_URL,
+} from "settings";
+
+import { getExtraLinkProps } from "utils/url";
+
+import "./MenuPopoverContent.scss";
+
+export const MenuPopoverContent: React.FC = () => (
+  <div className="MenuPopoverContent">
+    <a
+      href={TERMS_AND_CONDITIONS_URL}
+      {...getExtraLinkProps(true)}
+      className="MenuPopoverContent__link"
+    >
+      Terms of Use
+    </a>
+
+    <a
+      href={PRIVACY_POLICY}
+      {...getExtraLinkProps(true)}
+      className="MenuPopoverContent__link"
+    >
+      Privacy Policy
+    </a>
+
+    <a
+      href={HOMEPAGE_URL}
+      {...getExtraLinkProps(true)}
+      className="MenuPopoverContent__link"
```

Comment:
let's always move `...extra` type of props to the end. (3 / 3)
```suggestion
      className="MenuPopoverContent__link"
      {...getExtraLinkProps(true)}
```

Reply:
Let's not.

Always use judgement on case-by-case basis on how to order rest parameters with siblings. 
https://github.com/sparkletown/sparkle/pull/2010#discussion_r695412053